WordPress/gutenberg
# Site editor sidebar: home template details

## What?

Displays template and settings configuration details for the home and index templates.

See the designs and specs over at:
- https://github.com/WordPress/gutenberg/issues/49597

## Status

- Rest API changes for template/template part last modified in https://github.com/WordPress/gutenberg/pull/51362
- Clicking on a template area loads that template, but it breaks the back button. This is an existing issue, e.g., clicking on a template in the pages panel, then hitting back will take you to the templates panel (and not back to the pages panel)

## Testing Instructions
With a block theme activated, head over to the site editor and open the home or index template. 
Check that the settings controls work, and that the template area links open the correct template:
1. The posts per page and allow block comments controls, when saved, should be synched with the settings on `/wp-admin/options-reading.php`
2. When a static page is selected for the posts page on `/wp-admin/options-reading.php`, the post title control should display. Updating this field and saving should set the title of the posts page only.


## Screenshots or screencast 

State of this PR:

 | Home template as posts page  | Home template as Homepage |
| ------------- | ------------- |
| <img width="344" alt="Screenshot 2023-06-16 at 11 13 14 am" src="https://github.com/WordPress/gutenberg/assets/6458278/1a39ad18-643a-41d2-b403-3190614eb829">  | <img width="351" alt="Screenshot 2023-06-16 at 11 13 29 am" src="https://github.com/WordPress/gutenberg/assets/6458278/9dcb6f95-4450-4398-8c83-7ccffe7a9c0f"> |
